.codeclimate.yml

```diff
@@ -14,6 +14,8 @@ engines:
     checks:
       complexity:
         enabled: true
+      import/extensions:
```

Comment:
Mudar o .eslintrc resolveu o problema com `Missing file extension for "depedency"` mas o codeclimate continuou reclamando, desabilitando este check no codeclimate.

PS.: Aparentemente é um bug no plugin